attest/eventlog.go

```diff
@@ -31,6 +31,17 @@ import (
 	"github.com/google/go-tpm/tpmutil"
 )
 
+// ReplayError describes the parsed events that failed to verify against
+// a particular PCR.
+type ReplayError struct {
+	BadEvents   []Event
+	InvalidPCRs []int
```

Comment:
Events already contain the PCR index, no need to duplicate?

Reply:
Done. I've kept invalidPCRs around as unexported to keep the generation of the error string simple.